### PR TITLE
Add en summary & remove useless files

### PR DESCRIPTION
**Pull request by ysfscream**

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+      * [Data Point](products/product.md#datapoint)
+    * [Codec](products/codec.md)
+* [Security](device/security.md)
+  * [Certificate](device/certs.md)
+  * [One-way authentication](device/one_auth.md)
+  * [Mutual Authentication](device/two_auth.md)
+* [Timer](device/timer.md)
+
+## Rules
+* [Business rule](rule_engine/business_rules.md)
```
Comment (CrazyWisdom) [inline en/SUMMARY.md]: Business rules

File: en/SUMMARY.md [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+* [Scope rule](rule_engine/scope_rules.md)
```
Comment (CrazyWisdom) [inline en/SUMMARY.md]: Scope rules

File: en/SUMMARY.md [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+* [Actions](rule_engine/actions.md)
+
+## Alert
```
Comment (CrazyWisdom) [inline en/SUMMARY.md]: Alerts

File: en/SUMMARY.md [hunk 4]
```diff
@@ -1,5 +1,109 @@
...
+* [MQTT](test_center/mqtt.md)
+* [CoAP](test_center/coap.md)
+
+## Applications
+* [List](application/application.md#application-list)
+* [Role](application/application.md#application-role)
+
+## Users
+* [User](user/user.md#user)
+* [Role](user/user.md#role)
```
Comment (CrazyWisdom) [inline en/SUMMARY.md]: Roles

File: en/SUMMARY.md [hunk 5]
```diff
@@ -1,5 +1,109 @@
...
+## Test center
+* [MQTT](test_center/mqtt.md)
+* [CoAP](test_center/coap.md)
+
+## Applications
+* [List](application/application.md#application-list)
+* [Role](application/application.md#application-role)
+
+## Users
+* [User](user/user.md#user)
```
Comment (CrazyWisdom) [inline en/SUMMARY.md]: Users

**Comment by ysfscream** [comment]
@CrazyWisdom Fixed

**Pull request merged**